src/Tar/project.json

```diff
@@ -10,6 +10,7 @@
       "dependencies": {
         "NETStandard.Library": "1.6.0"
       }
-    }
+    },
+    "net46": {}
```

Comment:
I don't understand why this is needed... isn't this taken care of by the nestandard1.3 dependency above?